compiler/passes/scopeResolve.cpp

```diff
@@ -1747,6 +1747,27 @@ void checkConflictingSymbols(std::vector<Symbol *>& symbols,
   }
 }
 
+static void eliminateLastResortSyms(std::vector<Symbol*>& symbols) {
```

Comment:
How similar is this to the code in function resolution?  Would it be reasonable to share it?

Reply:
I don't think so because the function resolution runs the entire resolution process twice (once for not-last-resort and once only for last-resort). Now, of course, we could seek to share code between the two, but that would amount to rewriting the function resolution piece and isn't something I want to get into here.